extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.md

```diff
@@ -0,0 +1,131 @@
+
+# How it works
+
+1. Publishers can configure a Web Story with the component to implement a paywall.
+    - Paywall cannot be triggered on the first two pages of a Story or on the last page or on a Story with fewer than 4 pages.
+2. When the user navigates to the first paywalled page of the Story, <amp-story-subscriptions> will check if the user has access via <amp-subscriptions>. If yes, the user can consume the Story without any interruption.
+3. If the access is not granted, a paywall bottomsheet is shown at the bottom of the page. The paywall only prevents users from accessing the remaining pages of the Story, the user can otherwise navigate as normal (tap to previous pages, or swipe to a different Story if available).
+4. The paywall has a templated UX that allows for key messages to be customized by the publisher.
+5. Depending on publisher configuration, the dialog can support three different ways to progress:
+    - Sign-in with publisher’s sign-in flow (i.e. publisher provided sign-in page).
+    - Subscribe directly via the publisher (i.e. via a publisher provided subscription page).
+    - If enabled by the publisher, subscribe via [Subscribe with Google](https://developers.google.com/news/subscribe).
+6. If the user completes any of the above authentication flows, the user is returned to the Story page, the paywall is dismissed and the user can consume the rest of the Story.
+7. If the user encounters a subsequent paywalled Story from this publisher, they will continue to retain access based on [amp-subscription mechanisms](https://amp.dev/documentation/components/amp-subscriptions/#combining-the-amp-reader-id-with-publisher-cookies).
+
+# Subscriptions JSON Configuration Example
+
+```html
+...
+<script type="application/ld+json">
+     {
+       "@context": "http://schema.org",
+       "@type": "NewsArticle",
+       "isAccessibleForFree": "False",
+       "isPartOf": {
+         "@type": ["CreativeWork", "Product"],
+         "name" : "Product A",
+         "productID": "publisher.domain.com:product"
+       }
+     }
+</script>
+<script type="application/json" id="amp-subscriptions">
+     {
+       "services": [
+         {
+           "authorizationUrl": "https://publisher.domain.com/entitlement",
+           "pingbackUrl": "https://publisher.domain/pingback",
+           "actions":{
+             "login": "https://publisher.domain.com/signin",
+             "subscribe": "https://publisher.domain.com/subscribe"
+           }
+         },
+         {
+           "serviceId": "subscribe.google.com"
+         }
+       ]
+     }
+</script>
+...
+```
+
+See details about specifying this configuration in [`amp-subscriptions`](https://amp.dev/documentation/components/amp-subscriptions/).
```

Comment:
If there is no difference between the configuration for a Story or classic AMP, I think we could call it out specifically (you can port your existing configuration...) and just link to the doc without giving an example.

Reply:
Sure. Removed this section and call it out in the summary section.